rules/REQUEST-903.9003-NEXTCLOUD-EXCLUSION-RULES.conf

```diff
@@ -304,6 +304,7 @@ SecRule REQUEST_METHOD "@streq PUT" \
         "t:none,\
         ctl:ruleRemoveById=200002"
 
+
```

Comment:
See [this comment](https://github.com/SpiderLabs/owasp-modsecurity-crs/pull/1742#discussion_r412301157).